engine/src/gfx/coord_select.cpp

```diff
@@ -38,10 +37,11 @@ int CoordinateSelectChange = 0;
 int CoordinateSelectmousex;
 int CoordinateSelectmousey;
 extern Vector MouseCoordinate(int mouseX, int mouseY);
-extern KBSTATE keyState[LAST_MODIFIER][KEYMAP_SIZE];
+
+extern KBSTATE mouseButtonState;
```

Comment:
Um... what? Is it keyboard state or mouse button state?

At best, this is very confusing; at worst, it may not be correct at all.

Reply:
It's interesting....
https://wiki.libsdl.org/SDL2/SDL_GetMouseState - 32bit value
https://wiki.libsdl.org/SDL2/SDL_GetKeyboardState - 8bit value

Reply:
Checks for a modifier key is also pressed ie:Ctrl,Alt etc

Reply:
ahh...it's our own definition - https://github.com/vegastrike/Vega-Strike-Engine-Source/blob/master/engine/src/in.h#L25

Reply:
I like the `BUTTON_STATE` idea.

Reply:
I'm mixed on whether that should be in this PR or a separate one

Reply:
I don't want this in this PR. It's a mostly cosmetic thing and will impact 520 instances in around 30 files. It's better to do it in a separate PR.

Reply:
@royfalk OK, fair enough. That can be done in a separate PR.